packages/mantine/src/theme/Theme.tsx

```diff
@@ -93,12 +93,25 @@ export const plasmaTheme: MantineThemeOverride = {
                     color: theme.colors.gray[7],
                     marginBottom: theme.spacing.xs,
                 },
+                invalid: {
+                    color: theme.colors.red[9],
+                    borderColor: theme.colors.red[6],
+                },
+                error: {
+                    color: theme.colors.red[9],
+                },
             }),
         },
         TextInput: {
             defaultProps: {
                 radius: 8,
             },
+            styles: (theme) => ({
+                description: {
+                    fontSize: 'inherit',
+                    paddingBottom: theme.spacing.xs,
+                },
+            }),
```

Comment:
This change should not be in `InputWrapper` component to have consistency across all form components.